### PR TITLE
Added script to claim vested tokens and modified script to change owner and platformFeeTo for factory

### PR DESCRIPTION
**Pull request by xenide**

### CODE REVIEW

File: scripts/claimVestedTokens.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    }
+    else 
+    {
+        console.error("Invalid network specified");
+        process.exit(1);
+    }
+
+    if(!Web3.utils.isAddress(process.argv[3]))
+    {
+        console.error("Invalid address for vester contract");
```
Comment (OliverNChalk) [inline scripts/claimVestedTokens.js:38]: Interesting, so each user has their own personal vester contract, got it

**Review by OliverNChalk — APPROVED** [review]
Looks good, one thing to note is me & Ken will need to manually transfer the already accumulated platform fees to the DAO / Treasury address

File: scripts/claimVestedTokens.js (hunk 1 above)
Comment (xenide) [reply scripts/claimVestedTokens.js:38]: Correct, each TreasuryVester is designed to vest for only one address. 

Everyone can claim for themselves with the script, although it'd be public service if you do it for everyone from 0x15f

**Comment by xenide** [comment]
Yes, you guys will need to transfer the LP tokens to the timelock manually

**Review by leodsoft — APPROVED** [review]
all good

**Pull request merged**